### PR TITLE
Add weibo NER dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/weibo_ner/dataset_infos.json [hunk 1]
```diff
@@ -0,0 +1 @@
...name": "default", "version": {"version_str": "1.0.0", "description": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 1179589, "num_examples": 1350, "dataset_name": "turku_ner_corpus"}, "validation": {"name": "validation", "num_bytes": 232380, "num_examples": 270, "dataset_name": "turku_ner_corpus"}, "test": {"name": "test", "num_bytes": 237407, "num_examples": 270, "dataset_name": "turku_ner_corpus"}}, "download_checksums": {"https://raw.githubusercontent.com/OYE93/Chinese-NLP-Corpus/master/NER/Weibo/weiboNER_2nd_conll.train": {"num_bytes": 536039, "checksum": "4e8954c54ab33b0afc9a8abb85715c47422645cc6d1b49fdb24540f4a427e997"}, "https://raw.githubusercontent.com/OYE93/Chinese-NLP-Corpus/master/NER/Weibo/weiboNER_2nd_conll.dev": {"num_bytes": 105881, "checksum": "e72730d83aac680505280b6301c879b3ba470d29ac308979d48cdc58ee4907cc"}, "https://raw.githubusercontent.com/OYE93/Chinese-NLP-Corpus/master/NER/Weibo/weiboNER_2nd_conll.test": {"num_bytes": 108767, "checksum": "faaf6c0ebde5348b935cfd5c89f0f22c9e0779d60667d8b5efbc647b6707d50e"}}, "download_size": 750687, "post_processing_size": null, "dataset_size": 1649376, "size_in_bytes": 2400063}}
```
Comment (lhoestq) [inline datasets/weibo_ner/dataset_infos.json:1]: can you regenerate this file ? it looks like many fields are empty

File: datasets/weibo_ner/weibo_ner.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+ORG.NOM	泛指名称、统称（文艺公司）
+"""
+_HOMEPAGE_URL = "https://github.com/OYE93/Chinese-NLP-Corpus/tree/master/NER/Weibo"
+_CITATION = None
+_TRAIN_URL = "https://raw.githubusercontent.com/OYE93/Chinese-NLP-Corpus/master/NER/Weibo/weiboNER_2nd_conll.train"
+_TEST_URL = "https://raw.githubusercontent.com/OYE93/Chinese-NLP-Corpus/master/NER/Weibo/weiboNER_2nd_conll.test"
+_VALID_URL = "https://raw.githubusercontent.com/OYE93/Chinese-NLP-Corpus/master/NER/Weibo/weiboNER_2nd_conll.dev"
+
+
+class TurkuNERCorpus(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/weibo_ner/weibo_ner.py:38]:
```suggestion
class WeiboNer(datasets.GeneratorBasedBuilder):
```

**Review by lhoestq — COMMENTED** [review]
Nice thanks !

Could you fix the class name and then regenerate the dataset_infos.json file ?

File: datasets/weibo_ner/weibo_ner.py (hunk 2 above)
Comment (abhishekkrthakur) [reply datasets/weibo_ner/weibo_ner.py:38]: ahh. my copy-paste skills. will fix.!

**Review by lhoestq — APPROVED** [review]
LGTM !

**Pull request merged**